### ISSUE TITLE
Enable ``easy'' downloading of catalogs given the web path

### ISSUE THREAD

**Issue by nevencaplar** [astronomy-commons/lsdb#339]
Enable ``easy'' downloading of catalogs given the web path. 

For instance, something like

`lsdb.get_external_catalog('eypc_adress_goes_here', local_directory='', resume=True)`

with possibility to resume if it fails at some point.

**Comment by hombit** [comment]
I would question if it is LSDB's business to download catalogs.

If it is I propose `lsdb.download_catalog` name instead

**Comment by nevencaplar** [comment]
Yes, in conversations with Mario, this is a wanted functionality. The problem at the moment is that we offer catalogs and allow people to use them, but there is no easy way to get them. Without this functionality, we rely on relatively high buy-in (someone hipscat-importing large catalogs) or understanding they can do wget. If we want to support this behavior we need to make it simple.
